dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/Register/CheckAnswersTests.cs

```diff
@@ -128,6 +130,16 @@ public async Task Post_JourneyHasExpired_RendersErrorPage()
             authState.OnTrnLookupCompleted(trn, trnLookupStatus);
         }
 
+        TicketCreateRequest? ticketCreateRequestActual = null;
+        long ticketIdExpected = 1234567;
```

Comment:
Nit: this looks like a TRN, could we alter it a little to make it super clear?

Reply:
Sure